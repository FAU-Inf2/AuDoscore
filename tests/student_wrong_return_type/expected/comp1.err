ERROR - Method does not match with student counterpart: 
	 * student: public static long ToTest.toTest2()
	 *   clean: public static double ToTest.toTest2()
Exception in thread "main" java.lang.Error
	at tools.ic.InterfaceComparator.main(InterfaceComparator.java:336)
make[1]: *** [Makefile:146: run-comparer] Fehler 1
make: *** [Makefile:90: compile-stage1] Fehler 2
