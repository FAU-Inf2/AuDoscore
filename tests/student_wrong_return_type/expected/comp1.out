make[1]: Entering directory '/home/sitowert/AuDoscore/tests/student_wrong_return_type/test.2164'
rm -f *.class */*.class
javac -cp lib/junit5/junit-jupiter-api.jar:lib/junit5/junit-jupiter-engine.jar:lib/junit5/junit-jupiter-migrationsupport.jar:lib/junit5/junit-jupiter-params.jar:lib/junit5/junit-platform-commons.jar:lib/junit5/junit-platform-engine.jar:lib/junit5/junit-platform-launcher.jar:lib/junit5/junit-platform-reporting.jar:lib/junit5/junit-platform-runner.jar:lib/junit5/junit-platform-suite-api.jar:lib/junit5/junit-vintage-engine.jar:lib/apiguardian-api.jar:lib/opentest4j.jar:lib/junitpoints.jar -proc:only -processor tools.bomacon.BonusMalusConverter UnitTest.java > UnitTest.java.tmp ; mv UnitTest.java.tmp UnitTest.java
make[2]: Entering directory '/home/sitowert/AuDoscore/tests/student_wrong_return_type/test.2164'
javac -cp lib/json-simple-1.1.1.jar:lib/junit5/junit-jupiter-api.jar:lib/junit5/junit-jupiter-engine.jar:lib/junit5/junit-jupiter-migrationsupport.jar:lib/junit5/junit-jupiter-params.jar:lib/junit5/junit-platform-commons.jar:lib/junit5/junit-platform-engine.jar:lib/junit5/junit-platform-launcher.jar:lib/junit5/junit-platform-reporting.jar:lib/junit5/junit-platform-runner.jar:lib/junit5/junit-platform-suite-api.jar:lib/junit5/junit-vintage-engine.jar:lib/apiguardian-api.jar:lib/opentest4j.jar:lib/junitpoints.jar:. UnitTest.java ToTest.java 
make[2]: Leaving directory '/home/sitowert/AuDoscore/tests/student_wrong_return_type/test.2164'
java -cp lib/junit5/junit-jupiter-api.jar:lib/junit5/junit-jupiter-engine.jar:lib/junit5/junit-jupiter-migrationsupport.jar:lib/junit5/junit-jupiter-params.jar:lib/junit5/junit-platform-commons.jar:lib/junit5/junit-platform-engine.jar:lib/junit5/junit-platform-launcher.jar:lib/junit5/junit-platform-reporting.jar:lib/junit5/junit-platform-runner.jar:lib/junit5/junit-platform-suite-api.jar:lib/junit5/junit-vintage-engine.jar:lib/apiguardian-api.jar:lib/opentest4j.jar:lib/junitpoints.jar:. CheckAnnotation UnitTest
java -cp lib/junitpoints.jar:. ReadForbidden UnitTest > forbidden
chmod +x forbidden
if [ "x" != "x" ]; then \
	STUDENTCLASSES=$(find . -maxdepth 1 -iname "*.class" -a \! \( -name "UnitTest*" -o -name "*" \) ); \
else \
	STUDENTCLASSES=$(find . -maxdepth 1 -iname "*.class" -a \! -name "UnitTest*"); \
fi; \
if [ "x" != "x" ]; then \
	STUDENTCLASSES=$(echo "$STUDENTCLASSES" | grep -vE "$(echo "" | sed 's/\.java/(\\\$[^.]*)?\.class/g' | tr " " "|")"); \
fi; \
for i in $STUDENTCLASSES; do \
	rm -f javap.out; \
	javap -p -c $i > javap.out; \
	sed -i -e 's/(.*//' javap.out; \
	! ( cat javap.out | ./forbidden 1>&2 ) || exit 1; \
done
rm forbidden
make run-comparer
make[2]: Entering directory '/home/sitowert/AuDoscore/tests/student_wrong_return_type/test.2164'
java -cp lib/junitpoints.jar tools.ic.InterfaceComparer UnitTest
Makefile:112: recipe for target 'run-comparer' failed
make[2]: Leaving directory '/home/sitowert/AuDoscore/tests/student_wrong_return_type/test.2164'
Makefile:61: recipe for target 'compile-stage1' failed
make[1]: Leaving directory '/home/sitowert/AuDoscore/tests/student_wrong_return_type/test.2164'
